web/src/Web.App/Domain/CostCodes.cs

```diff
@@ -21,6 +23,29 @@ public class CostCodes(bool isPartOfTrust)
 
     // EducationalIctCostCodes
     private string LearningResourcesIctCostCode { get; } = isPartOfTrust ? "BAE210" : "E20";
+    private string ConnectivityCostCode { get; } = isPartOfTrust ? "" : "E20A";
+    private string OnsiteServersCostCode { get; } = isPartOfTrust ? "" : "E20B";
+    private string ItLearningResourcesCostCode { get; } = isPartOfTrust ? "" : "E20C";
+    private string LaptopsDesktopsAndTabletsCostCode { get; } = isPartOfTrust ? "" : "E20E";
+    private string OtherHardwareCostCode { get; } = isPartOfTrust ? "" : "E20F";
+    private string ItSupportCostCode { get; } = isPartOfTrust ? "" : "E20G";
+    private StringValues LearningResourcesIctCostCodes
+    {
+        get
+        {
+            return cfrItSpendBreakdown
+                ? new[]
+                {
+                    ConnectivityCostCode,
```

Comment:
Open question on work item: should `LearningResourcesIctCostCode` also be included here?